2021-09-17 14:36:42,778 - no description
2021-09-17 14:36:42,778 - dataset:ml-1m lr1:0.0005 lr2:0.00015 kl:0.05 contrast:0.1 batch:1 model_func:fc_att
2021-09-17 14:36:42,778 - Using CUDA:0
2021-09-17 14:36:44,531 - Building net...
2021-09-17 14:36:44,544 - Net build finished.
2021-09-17 14:36:44,544 - User datasets loaded and saved.
2021-09-17 14:36:44,545 - Start training...
2021-09-17 14:36:44,601 - Traceback (most recent call last):
  File "trainMe0917.py", line 423, in <module>
    train()
  File "trainMe0917.py", line 217, in train
    kl_loss = loss_func.kl_loss(adversary, x_real, z_inferred, padding, KL_WEIGHT=anneal)
  File "/home/lili/ACVAE-PyTorch-main/Federated_train1_GMM/loss_func.py", line 18, in kl_loss
    t_joint = adversary(x_real, z_inferred, padding)
  File "/home/lili/anaconda3/envs/vv/lib/python3.6/site-packages/torch/nn/modules/module.py", line 547, in __call__
    result = self.forward(*input, **kwargs)
  File "/home/lili/ACVAE-PyTorch-main/Federated_train1_GMM/modelMe0916.py", line 392, in forward
    net = net + 0.5 * torch.square(z)
AttributeError: module 'torch' has no attribute 'square'

2021-09-17 14:36:44,601 - Error.
